lib/daru/index/index.rb

```diff
@@ -222,6 +222,44 @@ def reorder(new_order)
       self.class.new(new_order.map { |i| from[i] })
     end
 
+    # Sorts a `Index` according to its values. Defaults to ascending order sorting.
+    # raises ArgumentError when all the values are not of the same class.
+    #
+    # == Options
+    #
+    # * +:ascending+ - if false, will sort in descending order. Defaults to true.
```

Comment:
I will edit the PR. Actually I was following these [syntax](https://github.com/SciRuby/daru/blob/master/lib/daru/vector.rb#L161)